PowerPointLabs/Test/FunctionalTest/ConvertToPictureTest.cs

```diff
@@ -66,12 +66,18 @@ private static void CheckIfClipboardIsRestored()
 
             // Paste whatever in clipboard
             Microsoft.Office.Interop.PowerPoint.ShapeRange newShape = actualSlide.Shapes.Paste();
+            // Set the shape location to 0 because the location of the shape being pasted is flaky
+            newShape.Top = 0;
```

Comment:
I think its better to set newshape position to copied position, and justify this "cheating" with the flakiness comment. Otherwise in case this method is used multiple times on a single slide, would be hard to differentiate the shapes

Reply:
Ok :)

Reply:
Good point! I have edited the test.